site/src/util/workspace.test.ts

```diff
@@ -40,4 +40,15 @@ describe("util > workspace", () => {
       expect(isWorkspaceOn(workspace)).toBe(isOn)
     })
   })
+  describe("workspaceQueryToFilter", () => {
+    it.each<[string | undefined, TypesGen.WorkspaceFilter]>([
+      [undefined, { Owner: "", OrganizationID: "" }],
+      ["", { Owner: "", OrganizationID: "" }],
+      ["asdkfvjn", { Owner: "", OrganizationID: "" }],
+      ["owner:me", { Owner: "me", OrganizationID: "" }],
+      ["owner:me owner:me2", { Owner: "me2", OrganizationID: "" }],
+    ])(`query=%p, filter=%p`, (query, filter) => {
+      expect(workspaceQueryToFilter(query)).toBe(filter)
```

Comment:
```suggestion
      expect(workspaceQueryToFilter(query)).toEqual(filter)
```

I know this is odd to imagine, but you can only use `toBe` on primitives. To compare objects, you need `toEqual` which checks deep equality but not referential equality.